roles/mesos/defaults/main.yml

```diff
@@ -8,7 +8,8 @@ mesos_agent_package: "mesos-agent-{{ mesos_version }}"
 mesos_mode: follower
 mesos_log_dir: /var/log/mesos
 mesos_logging_level: WARNING
-mesos_work_dir: /var/lib/mesos
+mesos_work_dir: /mnt/mesos
```

Comment:
@ryane I think using a single volume for Mesos and data frameworks would be fine. We should either
 1. Have different LVs for each component
 2. Have one LV for all components
 3. Have a documented rule for deciding which components have their own LVs and which ones share. 

But any of the above are fine with me. The simplest is definitely (2).